import-scripts/import_scripts/channel.py

```diff
@@ -521,10 +528,11 @@ def get_options(evaluation):
     options = get_options_raw(evaluation)
 
     @functools.lru_cache(maxsize=None)
+    @backoff.on_exception(backoff.expo, subprocess.CalledProcessError)
     def jsonToNix(value):
         result = subprocess.run(
             shlex.split(
-                "nix-instantiate --eval --strict -E 'builtins.fromJSON (builtins.readFile /dev/stdin)'"
+                "nix-instantiate --eval --strict -E 'builtins.fromJSON (builtins.areadFile /dev/stdin)'"
```

Comment:
Is this a typo?

Reply:
oh right, good catch. this was to test if it is failing correctly.